### PR TITLE
Update test-code.py

### PR DESCRIPTION
**Pull request by N8python**
# Pull Request
Add testLogDiv function.
## Description

  
At RDIL's request.

### CODE REVIEW

File: test-code.py [hunk 1]
```diff
@@ -105,7 +105,13 @@ def test_make_div() -> None:
         print("\n[DEBUG] make-div test did not fail")
     else:
         raise RuntimeError("make-div tests failed")
-
+        
+def testLogDivider() -> None:
+    print("[DEBUG] Running logDivider tests")
+    if not logDivider("---", "Hello") == "Hello":
```
Comment (RDIL) [inline test-code.py]: Question - what is the point of the "---" being passed?

**Comment by N8python** [comment]
To show how the function omits the divider if there is only one argument.

**Comment by RDIL** [comment]
Can you add a test also in the same function you just created for more than 1 arg?

**Comment by RDIL** [comment]
or even better just add it as an `and` in the if statement

**Comment by N8python** [comment]
sure

**Comment by RDIL** [comment]
you can merge if you want

**Comment by N8python** [comment]
Yay!

**Pull request merged**